airbyte-integrations/connectors/source-open-exchange-rates/integration_tests/configured_catalog.json

```
@@ -0,0 +1,1058 @@
+                "VES": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "VND": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "VUV": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "WST": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "XAF": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "XAG": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "XAU": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "XCD": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "XDR": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "XOF": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "XPD": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "XPF": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "XPT": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "YER": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "ZAR": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "ZMW": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                },
+                "ZWL": {
+                  "type": [
+                    "null",
+                    "number"
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "supported_sync_modes": [
+          "full_refresh",
+          "incremental"
+        ],
+        "source_defined_cursor": true,
+        "default_cursor_field": [
+          "timestamp"
+        ]
+      },
+      "sync_mode": "incremental",
+      "destination_sync_mode": "overwrite",
```

I think append would make more sense than overwrite with incremental!

```suggestion
      "destination_sync_mode": "append",
```